docs/CC2023/PAD.md

```diff
@@ -5,7 +5,7 @@
 
 ### Broad Description
 
-This artifact provides all the necessary code, scripts and results to compile the NMODL transpiler with the MOD2IR extension and run all benchmarks described in the manuscript. To simplify the evaluation process we provide along with the instructions a Dockerfile that will setup a viable system for the benchmarks. The driver script compiles the membrane mechanism model `hh.mod` and the synapse mechanism model `expsyn.mod` with various compile-time configurations and then runs the generated binaries comparing their runtimes. More specifically the benchmark compares the execution runtime of the binaries generated via the two-step compilation process MOD-C++-binary using various open-source and commercial compiler frameworks with the one-step ahead-of-time and just-in-time processes of MOD2IR. MOD2IR is implemented as a code generation backend inside the NMODL Framework and it makes heavy use of the LLVM IR and compilation passes. Most of the relevant code of the described work can be found [here](https://github.com/BlueBrain/nmodl/tree/llvm/src/codegen/llvm) and [here](https://github.com/BlueBrain/nmodl/tree/llvm/test/benchmark). The instructions to reproduce the results can be found [here](https://github.com/BlueBrain/nmodl/blob/896ffd002163ceb7c78f592456f31f66d314763a/docs/CC2023/README.md).
+This artifact provides all the necessary code, scripts and results to compile the NMODL transpiler with the MOD2IR extension and run all benchmarks described in the manuscript. To simplify the evaluation process we provide along with the instructions a Dockerfile that will setup a viable system for the benchmarks. The driver script compiles the membrane mechanism model `hh.mod` and the synapse mechanism model `expsyn.mod` with various compile-time configurations and then runs the generated binaries comparing their runtimes. More specifically the benchmark compares the execution runtime of the binaries generated via the two-step compilation process MOD-C++-binary using various open-source and commercial compiler frameworks with the one-step ahead-of-time and just-in-time processes of MOD2IR. MOD2IR is implemented as a code generation backend inside the NMODL Framework and it makes heavy use of the LLVM IR and compilation passes. Most of the relevant code of the described work can be found [here](https://github.com/BlueBrain/nmodl/tree/llvm/src/codegen/llvm) and [here](https://github.com/BlueBrain/nmodl/tree/llvm/test/benchmark). The instructions to reproduce the results can be found [here](https://github.com/BlueBrain/nmodl/blob/5bfd74850fb4c7346833d2ac0dfb001101b0e9fe/docs/CC2023/README.md).
```

Comment:
The commit id here needs to be updated once this PR is merged because there will be changes to README.md